deploy/config/keter.yaml

```diff
@@ -1,8 +1,9 @@
 stanzas:
-  - type: webapp
+  - type: background
```

Comment:
It looks like `keter` is running this `ensureAlive` function:

https://github.com/snoyberg/keter/blob/14d1e9e47f8d09ec3bbfc107d01187ea015587d2/Keter/App.hs#L326

This function appears to be just creating a TCP connection to `127.0.0.1:$PORT`.  If the connection fails for more than 90 seconds, then keter throws the `ensureAlive failed` message.

https://hackage.haskell.org/package/network-2.6.3.1/docs/Network.html#v:connectTo

I'm not sure why it is not working for our app.  Maybe our app does something strange when getting a TCP connection that is immediately closed.